z3c/dependencychecker/tests/test_report.py

```diff
@@ -225,6 +290,37 @@ def test_unneeded_requirements(capsys, minimal_structure):
     assert 'two' in out
 
 
+def test_unneeded_requirements_with_ignored_packages(
+    capsys,
+    minimal_structure
+):
```

Comment:
This `):` is in a weird location. It is apparently legal, but I guess it looks better if you place it on the line above.

Reply:
I really like to keep lines clean and not put everything and the kitchen sink there. Specially for future changes, if we need to add another parameter, a diff adding a single line would be enough, if we keep the parenthesis on it, we will have to change two lines.

But as it is a test and I want this to be merged and released I'm changing it :wink:

Reply:
I had to indent it *twice* (i.e 8 spaces) if not code analysis was complaining :)

Reply:
Ah, the "indentation continued on the same line" or so message? That is about the only pep8 warning that I dislike.

Reply:
Hm, there's something weird. I see that you've changed it, but I don't see the commit. And if I click on the original commit github can't find anything and gives a message "this sometimes happens with a forced push".

Did you do anything creative with git? :-)

There are other functions in that testfile with the same indentation style: leave them be. It might just be something I've got to get used to.